manifests/windows/install.pp

```diff
@@ -84,7 +84,7 @@
   # PUP-5480/PE-15037 Cache dir loses inheritable SYSTEM perms
   exec { 'fix inheritable SYSTEM perms':
     command => "${::system32}\\icacls.exe \"${::puppet_client_datadir}\" /grant \"SYSTEM:(OI)(CI)(F)\"",
-    unless  => "${::system32}\\icacls.exe \"${::puppet_client_datadir}\" | findstr \"SYSTEM:(OI)(CI)(F)\"",
+    unless  => "${::system32}\\cmd.exe /c ${::system32}\\icacls.exe \"${::puppet_client_datadir}\" | findstr \"SYSTEM:(OI)(CI)(F)\"",
```

Comment:
Hmm, now that I look at it again, does `cmd /c` require the rest of the command line to be passed as a quoted argument? I think `bash -c` does, but I'm not sure how similar Windows is...

Reply:
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd no, it is not _required_ to pass the command inside quotes:

"If you specify /c or /k, cmd processes the remainder of String"

However, you can use the `/s` switch to force the issue and wrap the command in quotes if you want. Using /s can make interpolation a little easier, since any quotes inside the command will be preserved.

TBH I don't care which format we use.

Reply:
Cool beans, that's good enough for me!